searchcore/src/vespa/searchcore/proton/common/alloc_strategy.h

```diff
@@ -0,0 +1,40 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include <vespa/searchcommon/common/compaction_strategy.h>
+#include <vespa/searchcommon/common/growstrategy.h>
+#include <iosfwd>
+
+namespace proton {
+
+/*
+ * Class representing allocation strategy
```

Comment:
Consider adding some details on what kind of allocations we are talking about.

Reply:
Class comment has been adjusted.